src/Backend/OnboardingInsuranceAPI/Services/UserInfo.cs

```diff
@@ -11,30 +12,17 @@ namespace OnboardingInsuranceAPI.Services;
 public class UserInfo
 {
     [Key]
-    [JsonProperty("pid")]
     public string Pid { get; set; }
-    [JsonProperty("name")]
     public string Name { get; set; }
-    [JsonProperty("surname")]
     public string Surname { get; set; }
-    [JsonProperty("birthdate")]
     public string Birthdate { get; set; }
-    [JsonProperty("birthNumber")]
     public string BirthNumber { get; set; }
-    [JsonProperty("mobileNumber")]
     public string MobileNumber { get; set; }
-    [JsonProperty("email")]
     public string Email { get; set; }
-    [JsonProperty("address1")]
     public string Address1 { get; set; }
-    [JsonProperty("address2")]
     public string Address2 { get; set; }
-    [JsonProperty("profileImage")]
     public string ProfileImage { get; set; }
-    [JsonProperty("carInsurance")]
     public CarInsuranceInfo CarInsurance { get; set; }
```

Comment:
This is not correct. As we said before: You cannot have the same model for DB and API. Even models used within models.

Reply:
Yeah, ok I'll remapp this.

Reply:
Same thing as the two models below.